lumigator/frontend/src/components/pages/LDatasets.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <div class="l-datasets">
+    <div class="l-datasets__list-container">
+      <ul v-if="datasets.length"
```

Comment:
should this be `datasets && datasets.length` ? to prevent errors during initial render?

Reply:
datasets.length is enough because of the `destructuring` using the `storeToRefs` function.
On Mounted hook we check the first part (if datasets!==null || undefined) by calling the `loadDatasets()`

Reply:
But without what I suggested I get this error:
`Uncaught (in promise) TypeError: $setup.datasets is undefined`

Reply:
Sounds like the store hasn't been setup properly. 
<img width="337" alt="image" src="https://github.com/user-attachments/assets/ae26e98a-79db-4db8-b3bd-ac98c73e7d96">


In any case the empty datasets are handled properly in the following [branch](https://github.com/mozilla-ai/lumigator/tree/LUMI-214_datasets-display) by the DataTable component.

Reply:
If I do a clean pull and I run `npm install`  and `npm run dev` I run into that issue not sure what's going on with the store or whatever but I don't think it should raise that error